prow/proxy-postsubmit.sh

```diff
@@ -29,6 +29,7 @@ set -u
 # Print commands
 set -x
 
+gcloud auth activate-service-account --key-file=/etc/service-account/service-account.json
```

Comment:
Use the `${GOOGLE_APPLICATION_CREDENTIALS}` env var instead of hard coding the location.